cfme/ansible/repositories.py

```diff
@@ -40,12 +40,7 @@ class RepositoryDetailsView(RepositoryBaseView):
     refresh = Button(title="Refresh this page")
     configuration = Dropdown("Configuration")
     download = Button(title="Download summary in PDF format")
-
-    @View.nested
-    class entities(View):  # noqa
-        properties = SummaryTable("Properties")
-        relationships = SummaryTable("Relationships")
-        repository_options = SummaryTable("Repository Options")
+    entity = ParametrizedView.nested(ParametrizedSummaryTable)
```

Comment:
I think I would prefer a consistent implementation of the table widget structure, and the `summary` name.  `view.entities.summary('title')` is a good call structure, if we can keep consistency.